lib/BracketingNonlinearSolve/test/rootfind_tests.jl

```diff
@@ -74,15 +75,27 @@ end
             @test result_tol > ϵ
         end
     end
+end
+
+@testitem "Zero-Tolerance Tests Interval Methods" setup=[RootfindingTestSnippet] tags=[:core] begin
+    prob=IntervalNonlinearProblem(quadratic_f, (1.0, 20.0), 2.0)
+    prob_lin=IntervalNonlinearProblem(linear_f, (-1.0, 1.0), 0.0)
 
-    # Some solvers support abstol=0.0 and converge to floating point precision
-    @testset for alg in (Bisection(), ITP(), Brent())
+    @testset for alg in (ITP(),)
```

Comment:
Why were the other methods removed from this? I would assume you'd expand this now?